packages/components/src/internal/components/search/utils.ts

```diff
@@ -718,6 +727,17 @@ export function isValidFilterFieldExcludeLookups(
     return !field.isLookup();
 }
 
+export function isValidFilterFieldSampleFinder(
```

Comment:
General grid filtering does work for `multiValue` fields, so I don't think we want this check in `isValidFilterField`

For the units column grid filtering, are you seeing errors or just getting unexpected results in the grid? This seems to be working for me locally.

Reply:
i am not seeing errors but I'm seeing unexpected results. If a sample doesn't have a unit (item.units is blank), the sample type's unit will be displayed. However, filtering filters on item.units, instead of the displayed value.